src/LEGO.AsyncAPI/Models/AsyncApiSecurityScheme.cs

```diff
@@ -9,6 +9,92 @@ namespace LEGO.AsyncAPI.Models
 
     public class AsyncApiSecurityScheme : IAsyncApiSerializable, IAsyncApiExtensible
     {
+        public static AsyncApiSecurityScheme UserPassword(string description = null) => new()
+        {
+            Type = SecuritySchemeType.UserPassword,
+            Description = description,
+        };
+
+        public static AsyncApiSecurityScheme ApiKey(ParameterLocation @in, string description = null) => new()
+        {
+            Type = SecuritySchemeType.ApiKey,
+            Description = description,
+            In = @in,
+        };
+
+        public static AsyncApiSecurityScheme X509(string description = null) => new()
+        {
+            Type = SecuritySchemeType.X509,
+            Description = description,
+        };
+
+        public static AsyncApiSecurityScheme SymmetricEncryption(string description = null) => new()
+        {
+            Type = SecuritySchemeType.SymmetricEncryption,
+            Description = description,
+        };
+
+        public static AsyncApiSecurityScheme AsymmetricEncryption(string description = null) => new()
+        {
+            Type = SecuritySchemeType.AsymmetricEncryption,
+            Description = description,
+        };
+
+        public static AsyncApiSecurityScheme HttpApiKey(string name, ParameterLocation @in, string description = null) => new()
+        {
+            Type = SecuritySchemeType.HttpApiKey,
+            Description = description,
+            Name = name,
+            In = @in,
+        };
+
+        public static AsyncApiSecurityScheme Http(string scheme, string? bearerFormat = null, string description = null) => new()
+        {
+            Type = SecuritySchemeType.Http,
+            Description = description,
+            Scheme = scheme,
+            BearerFormat = bearerFormat,
+        };
+
+        public static AsyncApiSecurityScheme OAuth2(AsyncApiOAuthFlows flows, string[] scopes = null, string description = null) => new()
+        {
+            Type = SecuritySchemeType.OAuth2,
+            Description = description,
+            Flows = flows,
+            Scopes = scopes,
+        };
+
+        public static AsyncApiSecurityScheme OpenIdConnect(string openIdConnectUrl, string description = null) => new()
+        {
+            Type = SecuritySchemeType.OpenIdConnect,
+            Description = description,
+            OpenIdConnectUrl = new Uri(openIdConnectUrl, UriKind.RelativeOrAbsolute),
```

Comment:
MUST be in the form of an absolute URL.

Reply:
Take Uri directly instead of string, and make a check that it is indeed an absolute url. throw if it isnt.